training/distributed_training/pytorch/model_parallel/gpt2/train_gpt_simple.py

```diff
@@ -634,6 +657,8 @@ def should_record():
                 out_path = os.path.join(args.checkpoint_dir, base_path)
                 total_ckpts = total_steps // args.checkpoint_freq
 
+                delete_oldest_ckpt(args, delete_on_rank0_only=args.use_fsx > 0)
```

Comment:
This isn't already in the file elsewhere? We didn't have this support?

Reply:
It was, just has been moved from L663 to here.